src/main/java/io/netty/incubator/codec/http3/Http3RequestStreamValidationHandler.java

```diff
@@ -31,15 +31,16 @@
 final class Http3RequestStreamValidationHandler extends Http3FrameTypeValidationHandler<Http3RequestStreamFrame> {
     private enum State {
         Initial,
-        Started,
+        HEADERS,
+        DATA,
```

Comment:
@NiteshKant nit: either user camel-case or upper-case for all of them. I don't care which one as long as its consistent :)

Reply:
lolz, sorry should have noticed.